staging/src/k8s.io/sample-controller/main.go

```diff
@@ -18,18 +18,17 @@ package main
 
 import (
 	"flag"
-	"time"
-
 	kubeinformers "k8s.io/client-go/informers"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/klog/v2"
+	"k8s.io/sample-controller/pkg/signals"
+	"time"
```

Comment:
We don't need to adjust the `time`

Reply:
I didn't get this comment. 'time' is neeeded because of  pre-existing logic where default resync duration is specifed.

`	kubeInformerFactory := kubeinformers.NewSharedInformerFactory(kubeClient, time.Second*30)
	exampleInformerFactory := informers.NewSharedInformerFactory(exampleClient, time.Second*30)`

Reply:
Put "time" into the import block at the top, the one for the standard library.

Reply:
Ah! I get it now. Thank you @yangjunmyfm192085 and Patrick.

Reply:
Fixed in recent commit.